corehq/apps/fixtures/tests/test_upload.py

```diff
@@ -329,7 +333,7 @@ def get_workbook_from_data(self, headers, rows):
     def upload(self, rows):
         data = self.make_rows(rows)
         workbook = self.get_workbook_from_data(self.headers, data)
-        _run_fixture_upload(self.domain, workbook)
+        type(self).do_upload(self.domain, workbook)
```

Comment:
`do_upload` is a function assigned to the class. Accessing it as `self.do_upload` causes it to be "bound" to `self`, which causes `self` to be passed as the first (`domain`) argument.

Reply:
Hm, that doesn't match the [python documentation](https://docs.python.org/3/library/functions.html#staticmethod) (or my experience). Static methods should be able to be called fine on instances of a class rather than just the class itself.

I did a test of that myself
```
>>> class A:
...     @staticmethod
...     def b(var):
...         print(var)
... 
>>> A.b('hi')
hi
>>> A().b('hi')
hi
```